content/blog/2022-05-06-CML-runners-saving-models-2.md

```diff
@@ -0,0 +1,243 @@
+---
+title:
+  Training and saving models with CML on a dedicated AWS EC2 runner (part 2)
+date: 2022-05-06
+description:
+  Use CML to automatically retrain a model on a provisioned AWS EC2 instance and
+  export the model to a DVC remote storage on Google Drive.
+descriptionLong: |
+  We can use CML to automatically retrain models whenever data, model code,
```

Comment:
Do we want to start with a note and a link here that this is part two, part one is here ...